src/main/webapp/scripts/poller.js

```diff
@@ -47,22 +41,30 @@
    * @private
    */ 
   poll_() {
-    throw new Error('Unimplemented');
+    this.result_ = this.pollingFunction_();
+    /**
+     * Represents the handler returned by the setTimeout that continues
+     * to poll.
+     * @private {number}
+     */
+    this.setTimeoutIdOfPoll_ = setTimeout(() => {
```

Comment:
Rename setTimeoutIdOfPoll_ to setTimeoutId_

The "ofPoll_" part is superfluous